storm-core/src/jvm/org/apache/storm/cluster/StateStorage.java

```diff
@@ -38,16 +41,16 @@
  * may or may not cause a collision in "/path".
  * Never use the same paths with the *_hb* methods as you do with the others.
  */
-public interface ClusterState {
+public interface StateStorage {
```

Comment:
StateStorage refers to classes which are related to zookeeper. If you want complete abstraction over different state store implementation, you should remove zookeeper specific dependencies in the interface.